### ISSUE TITLE
Error while compiling HTTP example

### ISSUE THREAD

**Issue by milkywade** [chrisjoyce911/esp32FOTA#116]
Hi - using last version 0.2.5, trying to compile the following example:

https://github.com/chrisjoyce911/esp32FOTA/blob/master/examples/HTTP/HTTP.ino

against this fota

```
{
    "type": "test_1",
    "version": "1",
    "url": "http://192.168.1.100/ota/test_1-version-1.bin"
}
```

returns the following message on the terminal (using platform.io):

```
[353940][E][HTTPClient.cpp:257] beginInternal(): failed to parse protocol
[353940][E][HTTPClient.cpp:257] beginInternal(): failed to parse protocol
```

P.S. the command executed (once - resulting in two error lines) is
`bool updatedNeeded = esp32FOTA.execHTTPcheck();`

**Comment by milkywade** [comment]
taking the "s from the version number results in the same error

**Comment by milkywade** [comment]
Tried several changes from the HTTP example, with no success:

1. changing the include statement to (as per the example on the main github page)
`#include <esp32fota.hpp>`

2. changing the version to a string, with three sub-versions, in this line:
`esp32FOTA esp32FOTA("cave_1", "1.0.0");`

3. using the `esp32FOTA.handle();` shortcut in the `loop()` section

...any ideas ?

**Comment by tobozo** [comment]
fixed in [0.2.6](https://github.com/chrisjoyce911/esp32FOTA/releases/tag/v0.2.6), will propagate soon in the Arduino Library Manager
